WebdriverIO.md

```diff
@@ -186,6 +189,8 @@ It might be tempting to use the `buttons[0]`, `buttons[last]`, and `buttons[n]`
 
 Except for cases where an element selector is crafted dynamically, all element selectors should be at the top of a utility file. There should be no element selectors in suite files.
 
+Note: We need to use `await` prefix while using `$$` selector in order to avoid flakiness of tests.
```

Comment:
Also, mention the case where we don't need it.

Reply:
Done.